docs/assets/css/app.scss

```diff
@@ -13,6 +13,7 @@
 @import "node_modules/nhsuk-frontend/packages/components/button/button";
 
 @import "node_modules/prismjs/themes/prism";
+@import "docs/assets/css/code-colours";
```

Comment:
If this is overriding all prism colours, can line 15 just be deleted?

Reply:
It's not overriding all the colours and it's not just colours in the prism theme file so it's still needed.